tests/20-build.sh

```diff
@@ -36,7 +29,6 @@ CONTAINER2="$SINGULARITY_TESTDIR/container2"
 alias container_check="stest 0 singularity exec \"$CONTAINER\" true ; 
 stest 1 singularity exec \"$CONTAINER\" false ; 
 stest 0 singularity exec \"$CONTAINER\" test -f /.singularity.d/runscript ; 
-stest 0 singularity exec \"$CONTAINER\" test -f /.singularity.d/labels.json ; 
 stest 0 singularity exec \"$CONTAINER\" test -f /.singularity.d/env/01-base.sh ; 
```

Comment:
we should probably have a deffile with %labels and test for the file still. It would be wrong to not have it, in that case.

Reply:
That's true.  I'll add another test in.